R/model.R

```diff
@@ -523,12 +527,14 @@ model_emax_binary <- function(
 }
 
 #' @rdname model
+#' @param bound is the upper truncation bound of the exponential scale parameter \eqn{b_3}
 #' @section Exponential Binary:
-#'   \deqn{y \sim Binomial(n, f(d))}
-#'   \deqn{link(f(d)) = b_1 + b_2 * (exp(b_3 * d) - 1)}
+#'   \deqn{y \sim N(f(d), \sigma^2)}
```

Comment:
This is a binary function, so needs to retain the binomial likelihood and description.